### ISSUE TITLE
Cannot hide fields with just one tab

### ISSUE THREAD

**Issue by rungta** [vaersaagod/matrixmate#9]
We have a block `links` with just two fields (`heading` and `linkslist`), and I would like to hide one of those fields (`heading`) in a particular context. This is the configuration I’m using to try and achieve this:

```php
return [
    'fields' => [
        'nav' => [
            'globalSet:legalese' => [
                'types' => [
                    'links' => [
                        'tabs' => [[
                            'label' => 'Show this field',
                            'fields' => 'linkslist',
                        ]],
                        'defaultTabName' => false,
                    ],
                ],
            ],
        ],
    ],
];
```

But it does not seem to have the intended effect. Both fields are visible with no tabs rendered, and no JS errors in the console. If I remove `'defaultTabName' => false,` then sure enough, two tabs are rendered with one field each.

**Comment by mmikkel** [comment]
Hi @rungta – thanks for reporting!

If you create a single tab, MatrixMate will actually render _no_ tabs - which was a conscious decision on our end, because we assumed "tabs are pointless if you only have one". We didn't consider this use case though.

I think the fix for this is to have MatrixMate actually render that single tab (which will effectively hide your `heading` field w/ your current config), but without the actual tab in the block header (which represents the pointless part of having a single tab).

Additionally, I'll think about adding a `hiddenFields` property for the `types` config (which would work similarly to the `hiddenTypes` setting available to the `groups` config. That would make it way easier to hide particular fields, without having to resort to using `tabs` and `defaultTabName` at all (which, when used like this, feels like a hacky way to do it).

I'll update this issue once I have the time to take a stab at it :)

**Comment by rungta** [comment]
Thanks for the fix @mmikkel!

I'd resorted to an ugly hack by setting the `defaultTabName` to some whitespace.

**Comment by mmikkel** [comment]
Hi @rungta, I just published [MatrixMate v1.2.0](https://github.com/vaersaagod/matrixmate/releases/tag/v1.2.0), which changes how single tabs are rendered – so your original code should work.

Additionally, there's a new [`hiddenFields`](https://github.com/vaersaagod/matrixmate#hide-fields) setting available, so whenever you need to hide one or several fields in a type config, you don't have to resort to using the `tabs` feature at all (unless you actually need tabs, of course).
